modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss

```diff
@@ -7,4 +7,26 @@
 // acceptable amount of bloat for maintaining styles automatically.
 body {
   @extend .su-wysiwyg-text;
+
+  a[href^="mailto:"] {
+    &::after {
+      @include margin(null 0.1em null 0.25em);
+      @include padding(null 0.01em null null);
+      background-image: url("#{$su-image-path}/arrow-down.png");
```

Comment:
Something isn't compiling correctly with this path. 
![image](https://user-images.githubusercontent.com/7185045/100677685-6320dc00-3320-11eb-858c-56e8a3c07346.png)